### PR TITLE
WIP - Logo only user

### PR DESCRIPTION
**Pull request by mosabua**
@martint this is a simple approach with the starburst logo a second time just to demo

Let me know some users and I will add those and replace

Also .. it links from the home page to the users page on all logos now. And then on the users page the logos either dont link anywhere or they go to the testimonial and from there via the separate link to the external site.. 

Let me know what you think about this simple behavior.

### CODE REVIEW

**Comment by martint** [comment]
Looks good, but I'd make the logos on the user page that don't have a testimonial not link to anywhere. It's feels a little weird for the link to be a no-op. It makes it appear as if something were broken.

**Comment by mosabua** [comment]
Okay... thats going to be a bit more work .. since the layout is used on home page and user page.. will have to either break it up or do something else.

Anyway.. you can work on the list of companies to add in here and I can sort that out ... probably tomorrow though

**Comment by martint** [comment]
Ok, don't worry about it for now.

**Comment by mosabua** [comment]
Dont worry about the commits .. I will rejig them all .. just want to get it working as desired for now .. including images.. so @aliloney feel free to commit any image updates on top as new commit

**Comment by mosabua** [comment]
The current setup is workable imho. If I need to completely drop the link on the users page from the ones without testimonial I need to separate the two layouts.. I can do that too but only if you think we need to @martint .. just test it out .. and also please chime in @aliloney

File: _data/users.yml [hunk 1]
```diff
@@ -81,6 +102,15 @@
...
   url: https://resurface.io/
+- name: Salesforce
+  logo: /assets/images/logos/salesforce.jpg
+  # https://engineering.salesforce.com/big-data-big-decisions-690a00fad88d
+- name: Sentinel One
+  logo: /assets/images/logos/sentinelone.png
+  # https://sentinelone-tech.medium.com/how-sentinelone-optimizes-big-data-at-scale-b957b5fa7b8a
+- name: Shopify
+  logo: /assets/images/logos/shopify.png
+  # https://shopify.engineering/shopifys-data-science-engineering-foundations
```
Comment (bitsondatadev) [inline _data/users.yml:113]: Are these just placeholders or we're planning on using these blogs as testimonials? I'm speaking right now with Nick from Shopify and he's getting clearance from his boss. Should we wait to get a green light before we merge this?

**Pull request closed without merging**